src/fix_deposit.cpp

```diff
@@ -583,8 +591,10 @@ void FixDeposit::pre_exchange()
 
   // warn if not successful b/c too many attempts
 
-  if (!success && comm->me == 0)
-    error->warning(FLERR,"Particle deposition was unsuccessful");
+  if (warnflag && !success && comm->me == 0) {
+    error->warning(FLERR,"One or more particle depositions were unsuccessful");
+    warnflag = 0;
```

Comment:
I do find the nonstop warnings tiring, but would it make sense to reset `warnflag` to 1 during the setup phase so you get one warning per run command? Might be handy to know whether changes between run commands improved the success.

Reply:
i agree, thanks for catching that